CHANGELOG.md

```diff
@@ -1431,7 +1431,7 @@ August 16, 2018
 - Fix bug causing expired resources to be re-requested causing an infinite loop #4255
 - Fix problem where `MapDataEvent#isSourceLoaded` always returned false #4254
 - Resolve an issue where tiles in the source cache were prematurely deleted, resulting in tiles flickering when zooming in and out and  #4311
-- Make sure `MapEventData` is passed through on calls `Map#flyTo` #4342
+- Make sure `MapDataEvent` is passed through on calls `Map#flyTo` #4342
```

Comment:
Revert. We shouldn't change old CHANGELOG entries

Reply:
Thank you @asheemmamoowala – I've removed this change from the PR in 08a758b